### DISCUSSION TITLE
Question: rate limit on api server?

### DISCUSSION THREAD

**Discussion by liyimeng** [k3s-io/k3s#7234] [Q&A]
**Environmental Info:**
K3s Version:  1.25.6

Node(s) CPU architecture, OS, and Version: 

Ubuntu 22/x86_64

Cluster Configuration:

3 server 1 node
**Describe the bug:**

I deploy rancher on my k3s cluster, after running rancher for a while, I found rancher is not able access anymore. 
the symptom is reported as https://github.com/rancher/rancher/issues/41056

I doubt that k3s api server have rate limit in someway, throttle rancher from serving more requests. But I could not find any configuration information regarding on rate limit in k3s, anyone could kindly help?  
**Steps To Reproduce:**

- Installed K3s: 
- install rancher
- restarting rancher for a couple of times to see the problem early.
-  
**Expected behavior:**

rancher should continue to work
**Actual behavior:**

rancher fail to start and being block from access for a very long time
**Additional context / logs:**

**Comment by brandond** [comment]
Rancher doesn't support Kubernetes 1.25 at the moment. You should use Rancher with a compatible version of Kubernetes.

https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-1/

**Comment by liyimeng** [comment]
@brandond you are right. Not supported in official release, but I'm running v2.7.2-rc8, which is supposed to support k8s 1.25

**Reply by liyimeng** [reply]
Rancher now have official support for 1.25, and problem still there :(
